07.ls_object/ls.rb

```diff
@@ -45,7 +45,7 @@ def align_array_size(rows_number, files)
   if modulo_files.zero?
     files
   else
-    files + Array.new(rows_number - modulo_files, LsFile.new(directory_path: '', file_name: ''))
+    files + Array.new(rows_number - modulo_files, LsFile.new('', ''))
```

Comment:
ファイルシステム上には `''` というファイルが存在するわけではないので、nilで表現する方が自然かなと思います。

```suggestion
    files + Array.new(rows_number - modulo_files)
```

Reply:
改めて考えると違和感があったので、ご指摘いただいた通り修正いたしました 🙇